cfgmgr/buffermgr.cpp

```diff
@@ -264,7 +282,7 @@ task_process_status BufferMgr::doSpeedUpdateTask(string port)
             if ((fvField(prop) == "profile") && (profile_ref == fvValue(prop)))
             {
                 SWSS_LOG_NOTICE("PG to Buffer Profile Mapping %s already present", buffer_pg_key.c_str());
-                continue;
+                return task_process_status::task_success;
```

Comment:
we shouldn't return, we still need to check all lossless pgs. You can set a flag here and then outside this L280 loop, skip applying the pg if the flag is set

Reply:
Oh, I'm aware the low-level mistake just now. Please help check again. Thanks for pointing it out.